infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LocationTrackService.kt

```diff
@@ -500,46 +494,47 @@ class LocationTrackService(
     fun getSplittingInitializationParameters(
         locationTrackId: IntId<LocationTrack>,
         publishType: PublishType,
-    ): SplittingInitializationParameters {
+    ): SplittingInitializationParameters? {
```

Comment:
Pihvinä täällä tämä: Nämä voi nykyään palautua nulleina, niin bäkkäri ei heitä tilanteissa joissa näitä ei saada laskettua